source/docs/zero-to-robot/step-1/how-to-wire-a-robot.rst

```diff
@@ -167,11 +167,10 @@ Weidmuller Connectors
 .. raw:: html
 
     <div style="position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
-        <iframe src="https://www.youtube-nocookie.com/embed/kCcDw3lDYis" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe>
+        <iframe src="https://www.youtube-nocookie.com/embed/Zsnro-9_ZfQ" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe>
     </div>
 
-
-.. warning:: The correct strip length is ~5/16" (~8 mm), not the 5/8" (~16 mm) mentioned in the video.
+.. warning:: The recommended wire strip length is ~5/16" (~8 mm).
```

Comment:
I think the warning could be deleted. It was needed when the video gave the wrong value. However, the correct value is given below.

Reply:
Just updated it to remove the warning